python/cudf/cudf/core/dataframe.py

```diff
@@ -5866,8 +5866,16 @@ def to_struct(self, name=None):
         -----
         Note that a copy of the columns is made.
         """
+        if not all(isinstance(name, str) for name in self._data.names):
+            warnings.warn(
+                "DataFrame contains non-string column name(s). Struct column "
+                "requires field name to be string. Non-string column names will "
+                "be casted to string as the field name."
```

Comment:
This suggestion fixes flake8 line length limits so that style checks pass. I'll apply it now.
```suggestion
                "requires field name to be string. Non-string column names "
                "will be casted to string as the field name."
```